crates/bevy_reflect/src/func/function.rs

```diff
@@ -185,16 +188,29 @@ impl<'env> DynamicFunction<'env> {
     pub fn info(&self) -> &FunctionInfo {
         &self.info
     }
+
+    /// The [name] of the function.
+    ///
+    /// For [`DynamicFunctions`] created using [`IntoFunction`],
+    /// the name will always be the full path to the function as returned by [`std::any::type_name`].
+    /// This can be overridden using [`with_name`].
+    ///
+    /// [name]: FunctionInfo::name
+    /// [`DynamicFunctions`]: DynamicFunction
+    /// [`with_name`]: Self::with_name
+    pub fn name(&self) -> &Cow<'static, str> {
+        self.info.name()
+    }
 }
 
 /// Outputs the function signature.
 ///
 /// This takes the format: `DynamicFunction(fn {name}({arg1}: {type1}, {arg2}: {type2}, ...) -> {return_type})`.
 ///
-/// Names for arguments and the function itself are optional and will default to `_` if not provided.
+/// Names for arguments are optional and will default to `_` if not provided.
```

Comment:
This doc line seems out of date.

Reply:
Argument names are still optional since we can't infer these through the type system unfortunately :(